crates/pipeline-manager/src/config.rs

```diff
@@ -251,34 +259,41 @@ impl DatabaseConfig {
     /// Database connection string.
     fn database_connection_string(&self) -> String {
         if self.db_connection_string.starts_with("postgres") {
-            // this starts_with works for `postgres://` and `postgres-embed`
+            // this starts_with works for `postgres://`, `postgres-embed` and `postgres-pg-client-embed`
```

Comment:
technically you need a comma before the "and" as well

Reply:
Will fix it in the paper